docs/reference/aggregations/metrics/weighted-avg-aggregation.asciidoc

```diff
@@ -210,8 +210,7 @@ Which should look like:
 The `missing` parameter defines how documents that are missing a value should be treated.
 The default behavior is different for `value` and `weight`:
 
-By default, if the `value` field is missing the document is ignored and the aggregation moves on to the next document.
-If the `weight` field is missing, it is assumed to have a weight of `1` (like a normal average).
+By default, if the `value` or `weight` field is missing the document is ignored and the aggregation moves on to the next document.
 
 Both of these defaults can be overridden with the `missing` parameter:
```

Comment:
Some of the text surrounding this change is still incorrect. For example:

> The default behavior is different for `value` and `weight`:

is still wrong.

I left a suggestion that fixes up the surrounding text and shortens this a bit. I also updated the text to mention `null` values, which are treated the same as missing values based on my testing. Feel free to change the wording or let me know what you think.

```suggestion
By default, the aggregation excludes documents with a missing or `null` value
for the `value` or `weight` field. Use the `missing` parameter to specify a
default value for these documents instead.
```
```